### DISCUSSION TITLE
Can't setup local cpan library following directions in brew info perl

### DISCUSSION THREAD

**Discussion by mike-onehub** [Homebrew/discussions#42] [Everyday usage]
Error messages:
PERL_MM_OPT="INSTALL_BASE=$HOME/perl5" cpan local::lib
Loading internal logger. Log::Log4perl recommended for better logging
mkdir /Users/michael.gracy: Permission denied at /usr/local/Cellar/perl/5.32.0/lib/perl5/5.32.0/CPAN/FTP.pm line 547.

I've checked brew doctor. I got a couple of warnings, but nothing permissions related.
I reran a chown on usr/local just to be sure and still get the same error. I looked at that FTP.pm module and can't make sense of what is going on. So I'm hoping someone will have an idea. Thanks.

**Comment by SMillerDev** [comment]
It's trying to create `/Users/michael.gracy` I don't think this is an issue in brew. You should ask the FTP module developers why it's doing that.

**Comment by mike-onehub** [comment]
Sounds like a good idea. I will do that, thank you. I wish I had a better understanding of perl.
